extension/js/common/api/email-provider/gmail/google-auth.ts

```diff
@@ -176,25 +176,25 @@ export class GoogleAuth {
         };
       }
       try {
-        const potentialFes = new EnterpriseServer(authRes.acctEmail);
+        const potentialFes = new ExternalService(authRes.acctEmail);
         if (await potentialFes.isFesInstalledAndAvailable()) {
           // on FES, pulling ClientConfiguration is not authenticated, and it contains info about how to
           //   authenticate when doing other calls (use access token or OIDC directly)
           await AcctStore.set(authRes.acctEmail, { fesUrl: potentialFes.url });
           const acctServer = new AccountServer(authRes.acctEmail);
           // fetch and store ClientConfiguration (not authenticated)
-          await acctServer.accountGetAndUpdateLocalStore();
+          await acctServer.fetchAndSaveClientConfiguration();
         } else {
           // eventually this branch will be dropped once a public FES instance is run for these customers
-          // when using flowcrypt.com/api, pulling ClientConfiguration is authenticated, therefore have
+          // when using flowcrypt.com/shared-tenant-fes, pulling ClientConfiguration is authenticated, therefore have
           //   to retrieve access token first (which is the only way to authenticate other calls)
           const acctServer = new AccountServer(authRes.acctEmail);
           // fetch and store ClientConfiguration (authenticated)
-          await acctServer.accountGetAndUpdateLocalStore(); // stores ClientConfiguration
+          await acctServer.fetchAndSaveClientConfiguration(); // stores ClientConfiguration
```

Comment:
There is an opportunity to simplify here it seems, I think the two branches don't quite have to be two branches anymore, but unfortunately I can't provide exact details on what to do here without sitting down and trying to rewrite it myself. Please try taking a look.

Reply:
We could alternatively do it in another PR.. for safety.